IO/BeagleBone/BBBPinManager.cs

```diff
@@ -9,1355 +9,1367 @@
-                        Output.Add("        __overlay__ {");
-                        Output.Add("            status = \"okay\";");
-                        Output.Add("            pinctrl-names = \"default\";");
-                        Output.Add("            pinctrl-0 = <&scarlet_pwm2_pins>;");
-                        Output.Add("        };");
-                        Output.Add("    };");
-                        Output.Add("    ");
-                    }
-                }
-            }
-
-            // Output I2C device fragments
-            if(I2CMappings != null)
-            {
-                lock(I2CMappings)
-                {
-                    Output.Add("    fragment@3 {");
-                    Output.Add("        target = <&am33xx_pinmux>;");
-                    Output.Add("        __overlay__ {");
-                    if (I2CDev1.Count > 0)
-                    {
-                        Output.Add("            bbb_i2c1_pins: pinmux_bbb_i2c1_pins {");
-                        Output.Add("                pinctrl-single,pins = <");
-                        foreach (PinAssignment PinAss in I2CDev1.Values)
-                        {
-                            string Offset = String.Format("0x{0:X3}", (Pin.GetOffset(PinAss.Pin) - 0x800));
-                            string Mode = String.Format("0x{0:X2}", PinAss.Mode);
-                            Output.Add("                    " + Offset + " " + Mode);
-                        }
-                        Output.Add("                >;");
-                        Output.Add("            };");
-                    }
-                    if (I2CDev2.Count > 0)
-                    {
-                        Output.Add("            bbb_i2c2_pins: pinmux_bbb_i2c2_pins {");
-                        Output.Add("                pinctrl-single,pins = <");
-                        foreach (PinAssignment PinAss in I2CDev2.Values)
-                        {
-                            string Offset = String.Format("0x{0:X3}", (Pin.GetOffset(PinAss.Pin) - 0x800));
-                            string Mode = String.Format("0x{0:X2}", PinAss.Mode);
-                            Output.Add("                    " + Offset + " " + Mode);
-                        }
-                        Output.Add("                >;");
-                        Output.Add("            };");
-                    }
-                    Output.Add("        };");
-                    Output.Add("    };");
-                    Output.Add("    ");
-
-                    if (I2CDev1.Count > 0)
-                    {
-                        Output.Add("    fragment@20 {");
-                        Output.Add("        target = <&i2c1>;");
-                        Output.Add("        __overlay__ {");
-                        Output.Add("            status = \"okay\";");
-                        Output.Add("            pinctrl-names = \"default\";");
-                  
```

Comment:
Without knowing what was changed, we can't merge this.
You'll need to fix whatever is causing this entire file to be one changeset in order for me to be able to review.